src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,14 @@
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.AWSLambdaInstrumentationOptions() -> void
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.DisableAwsXRayContextExtraction.get -> bool
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.AWSLambdaInstrumentationOptions.DisableAwsXRayContextExtraction.set -> void
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation.AWSLambdaWrapper
+OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation.AWSLambdaWrapper.AWSLambdaWrapper() -> void
```

Comment:
This public ctor is actually unintentionally exposed, fixed in #593 after noticing through this line.